apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -122,6 +122,19 @@
 
 #define INV_FD -3
 
+#define DEV_PATH "/dev"
+#define DEV_INVALID_DIR "/dev/zero"
```

Comment:
please align these values.
if you define DEV_INVALID_DIR then it should be defined something like '/dev/invalid'

Reply:
Okay will change accordingly.